app/misc/git.ts

```diff
@@ -495,6 +495,37 @@ function createBranch() {
   }
 }
 
+// search for tags
+function searchTag() {
+  Git.Repository.open(repoFullPath)
+    .then(function (repo) {
+      repo.getCurrentBranch()
+        .then(function () {
+          // grab the list of references - these could be branches or tags
+          return repo.getReferences(Git.Reference.TYPE.LISTALL);
+        }).then(function (refList) {
+          refList.sort();
+          for (let i = 0; i < refList.length; i++) {
+
+            // strip name for readability
+            let refName = refList[i].name().split("/")[refList[i].name().split("/").length - 1];
+
+            if (refList[i].isTag()){
+              if (refName.indexOf( document.getElementById("tag-name").value ) > -1) {
```

Comment:
Just to be sure, if you find the refname in the tagname then it returns > -1?

Reply:
Yes, js has a built-in function which helps you compare strings. This will check if refName is a substring of the tag we're currently comparing to.